omnibus/resources/fips/openssl.cnf

```diff
@@ -0,0 +1,15 @@
+config_diagnostics = 1
+openssl_conf = openssl_init
+
+# TODO: template install path
+.include /opt/datadog-agent/embedded/ssl/fipsmodule.cnf
```

Comment:
This should probably handle https://github.com/DataDog/datadog-agent/pull/29121

Reply:
You're right, thanks 👍 67379db693939da5687e588b65a3aa82c02d2ffb